package.json

```diff
@@ -193,6 +192,7 @@
   },
   "sideEffects": false,
   "resolutions": {
-    "koa-router/path-to-regexp": "6.1.0"
+    "koa-router/path-to-regexp": "6.1.0",
+    "@types/react": "16.14.29"
```

Comment:
I'm not 100% how this works but do we need this or can we pin the version in the dependencies above?

Reply:
This forces any direct or transitive dependency to use specific version of package (see https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/)

Using resolutions is usually last resort for conflicting transitive dependencies.  This is exactly our case - storybook and friends all want @types/react ^18.something and we don't want to go back to very old storybook that wants react 16

Reply:
great, thanks :)